server/routes/settings/index.ts

```diff
@@ -231,11 +243,31 @@ settingsRoutes.get('/tautulli', (_req, res) => {
   res.status(200).json(settings.tautulli);
 });
 
-settingsRoutes.post('/tautulli', async (req, res) => {
+settingsRoutes.post('/tautulli', async (req, res, next) => {
   const settings = getSettings();
 
   Object.assign(settings.tautulli, req.body);
-  settings.save();
+
+  try {
+    const tautulliClient = new TautulliAPI(settings.tautulli);
+
+    const result = await tautulliClient.getInfo();
+
+    if (!semver.gte(semver.coerce(result?.tautulli_version) ?? '', '2.9.0')) {
```

Comment:
Do we really need a whole dependency added to perform this check? We can split on periods and check ourselves and remove the need to keep another dependency up to date.